### ISSUE TITLE
Update 1.7.3 to 1.8 seems to be broken - [E] InitIssuesIndexer: create index, mkdir /usr/local/bin/data: permission denied

### ISSUE THREAD

**Issue by mbedded** [go-gitea/gitea#6398]
- Gitea version (or commit ref): 1.8
- Git version: 2.11.0
- Operating system: Raspbian
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [X] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant
- Log gist:
```
2019/03/20 17:11:33 [I] Log Mode: File(Info)
2019/03/20 17:11:33 [I] XORM Log Mode: File(Info)
2019/03/20 17:11:33 [I] Cache Service Enabled
2019/03/20 17:11:33 [I] Session Service Enabled
2019/03/20 17:11:33 [I] Git Version: 2.11.0
2019/03/20 17:11:34 [...les/indexer/issue.go:64 InitIssueIndexer()] [E] InitIssuesIndexer: create index, mkdir /usr/local/bin/data: permission denied
```

## Description

I did an update from version 1.7.3 to 1.8. As the documentation says i have just to download the new file, name it "gitea" and restart my service. I am using Gitea on my raspberry

```
systemctl stop gitea.service
mv gitea gitea_bak
wget -O gitea https://dl.gitea.io/gitea/1.8/gitea-1.8-linux-arm-7
chmod +x gitea
systemctl start gitea.service
```

I have done this before to update the last 2 times. It works like a charm.

Previous updates were just fine. Everything works as before. As i updated from 1.7.3 to 1.8 i was unable to start the server by hand.
Unfortunately the log is not filled when i use `systemctl start gitea.service`. There i am only getting the status that the service is not running anymore.

```
gitea.service - Gitea on Raspberry
   Loaded: loaded (/etc/systemd/system/gitea.service; enabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: signal) since Wed 2019-03-20 17:32:35 CET; 77ms ago
  Process: 9333 ExecStart=/home/pi/gitea/gitea web -c /home/pi/gitea/custom/conf/app.ini (code=killed, sign
 Main PID: 9333 (code=killed, signal=ILL)

Mar 20 17:32:35 gitea01 systemd[1]: gitea.service: Unit entered failed state.
Mar 20 17:32:35 gitea01 systemd[1]: gitea.service: Failed with result 'signal'.
```

I am not using /usr/local/bin.. to store my data. I changed  no paths of my configuration.
When i replace the new gitea executable with the old version (1.7.3) i have no issues and can start the service by hand or service without modifying anything.

Could it be related to #5482 or #4672?

**Comment by focmb** [comment]
Same here on a raspberry. For gitea I use a seperate user with home folder (user: gitea, Home folder: /home/gitea). I store all gitea files in the home folder of gitea. Until version 1.7.3 all was working fine but with the new update (version 1.8.0) I get the same error like above.

PS: Instead of MySQL I use sqlite3.

**Comment by ChrisB85** [comment]
Probably related to https://github.com/go-gitea/gitea/issues/6367

**Comment by lafriks** [comment]
@lunny could this be related to your indexer refactoring?

**Comment by lunny** [comment]
@lafriks @mbedded there is no function named `InitIssueIndexer()` in `release/v1.8`. That should be on `release/v1.7` I think.

**Comment by mbedded** [comment]
@lunny Thanks for that information. I pasted the last lines of my `gitea.log` file. Maybe this was an old entry because i tried to create/get the same issue with an older version of gitea so i started and stopped the service several times.

Maybe this ticket can be closed as it seems to be a duplicate of #6367. I was looking for existing issues but didn't found any. I guess i was searching with the wrong keywords.

**Comment by lunny** [comment]
duplicated with #6367
